### PR TITLE
added support for tpopes repeat

### PR DESCRIPTION
**Pull request by Gavinok**
I added support for tpopes [repeat.vim](https://github.com/tpope/vim-repeat) plugin to <kbd>\<C-X\></kbd>, <kbd>\<C-A\></kbd>, and  <kbd>cic</kbd> .

### CODE REVIEW

File: ftplugin/dotoo.vim [hunk 1]
```diff
@@ -32,11 +32,15 @@ autocmd! BufWritePost <buffer> call dotoo#parser#parsefile({'force': 1})
...
-nnoremap <buffer> <silent> gI :<C-U>call dotoo#clock#start()<CR>
-nnoremap <buffer> <silent> gO :<C-U>call dotoo#clock#stop()<CR>
-nnoremap <buffer> <silent> gM :<C-U>call dotoo#move_headline(dotoo#get_headline())<CR>
-nnoremap <buffer> <silent> cit :<C-U>call dotoo#change_todo()<CR>
-nnoremap <buffer> <silent> cic :<C-U>call dotoo#checkbox#toggle()<CR>
-nnoremap <buffer> <silent> <C-A> :<C-U>call dotoo#increment_date()<CR>
-nnoremap <buffer> <silent> <C-X> :<C-U>call dotoo#decrement_date()<CR>
+nnoremap <silent> <Plug>DotooIncrementDate  :<C-U>call dotoo#increment_date(v:count1)<CR>
+nnoremap <silent> <Plug>DotooDecrementDate  :<C-U>call dotoo#decrement_date(v:count1)<CR>
+nnoremap <silent> <Plug>DotooCheckboxToggle :<C-U>call dotoo#checkbox#toggle()<CR>
```
Comment (dhruvasagar) [inline ftplugin/dotoo.vim]: I think we should move these `<Plug>` mappings to `plugin/tablemode.vim` and in the ftplugins.

File: ftplugin/dotoocapture.vim [hunk 2]
```diff
@@ -18,5 +18,8 @@ augroup BufWrite
 
   autocmd BufHidden <buffer> call s:RefileAndClose()
 augroup END
-nnoremap <buffer> <silent> <C-A> :<C-U>call dotoo#increment_date()<CR>
-nnoremap <buffer> <silent> <C-X> :<C-U>call dotoo#decrement_date()<CR>
+
+nnoremap <silent> <Plug>DotooIncrementDate :<C-U>call dotoo#increment_date(v:count1)<CR>
+nnoremap <silent> <Plug>DotooDecrementDate :<C-U>call dotoo#decrement_date(v:count1)<CR>
```
Comment (dhruvasagar) [inline ftplugin/dotoocapture.vim]: Same as above, also no reason why checkbox toggling shouldn't be available while capturing.

File: ftplugin/dotoo.vim (hunk 1 above)
Comment (Gavinok) [reply ftplugin/dotoo.vim]: I assume that you mean plugin/dotoo.vim makes sense just changed this

Comment (dhruvasagar) [reply ftplugin/dotoo.vim]: Oops, sorry but yes

**Pull request merged**